itest/consumer_test_manager.go

```diff
@@ -1,38 +1,111 @@
 package e2etest
 
 import (
+	"path/filepath"
 	"strconv"
 	"strings"
 	"testing"
 
+	sdklogs "cosmossdk.io/log"
 	sdkmath "cosmossdk.io/math"
+	wasmapp "github.com/CosmWasm/wasmd/app"
+	wasmparams "github.com/CosmWasm/wasmd/app/params"
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
 	bbntypes "github.com/babylonchain/babylon/types"
+	fpcc "github.com/babylonchain/finality-provider/clientcontroller"
+	"github.com/babylonchain/finality-provider/eotsmanager/client"
+	eotsconfig "github.com/babylonchain/finality-provider/eotsmanager/config"
+	"github.com/babylonchain/finality-provider/finality-provider/config"
+	fpcfg "github.com/babylonchain/finality-provider/finality-provider/config"
 	"github.com/babylonchain/finality-provider/finality-provider/service"
+	"github.com/babylonchain/finality-provider/types"
+	"github.com/btcsuite/btcd/btcec/v2"
+	dbm "github.com/cosmos/cosmos-db"
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
 )
 
 type ConsumerTestManager struct {
-	*TestManager
-	WasmdHandler *WasmdNodeHandler
+	BabylonHandler      *BabylonNodeHandler
+	FpConfig            *fpcfg.Config
+	BBNClient           *fpcc.BabylonController
+	WasmdHandler        *WasmdNodeHandler
+	WasmdConsumerClient *fpcc.WasmdConsumerController
```

Comment:
Not for this PR: later would these two fields be generalised to sth like

```suggestion
	consumerHandler        ConsumerHandler
	consumerClient         fpcc.ConsumerController
```

where `ConsumerHandler` and `fpcc.ConsumerController` are interfaces?

Reply:
sure, actually i have renamed the files so its like this now

`WasmdConsumerClient *fpcc.CosmwasmConsumerController`

the controller is named to represent cosmwasm enabled chains, on the left side the variable is still WasmdConsumerClient because in the tests we are using wasmd as the consumer chain.